com.obsproject.Studio.Plugin.TransitionTable.metainfo.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>com.obsproject.Studio.Plugin.TransitionTable</id>
+  <extends>com.obsproject.Studio</extends>
+  <name>Transition Table Plugin</name>
+  <summary>Plugin for OBS Studio to add a Transition Table to the tools menu</summary>
```

Comment:
Hm, how about something like `Fine-tune your transitions with a transition table`?

(I've been advocating for removing things like "Plugin for OBS Studio that ..." from summaries. Instead, I'm suggesting summaries that briefly explain what that plugin enables people to do.)